src/runway.rs

```diff
@@ -619,9 +617,12 @@ where
             NotificationOut::AddedToDag(h, p_hashes) => {
                 self.store.add_parents(h, p_hashes);
                 self.resolve_missing_parents(&h);
-                let coord = self.store.unit_by_hash(&h).map(|u| u.as_signable().coord());
-                if let Some(coord) = coord {
-                    self.resolve_missing_coord(&coord)
+                if let Some(su) = self.store.unit_by_hash(&h).cloned() {
+                    let coord = su.as_signable().coord();
```

Comment:
This unit does not have a `creator` method?

Reply:
It does, nice catch!